db/migrate/20241024204511_create_interfaces.rb

```diff
@@ -0,0 +1,10 @@
+class CreateInterfaces < ActiveRecord::Migration[7.0]
+  def change
+    create_table :interfaces do |t|
+      t.string :external_id
+      t.integer :interface_type, null: false
```

Comment:
if we are on the `interfaces` table, there is no need to name the column `inteface_type`. I'd prefer to call it `type`. WDYT?

Reply:
It's a problem, look:

<img width="1345" alt="image" src="https://github.com/user-attachments/assets/0cb999c0-143d-4130-b3c5-c7a07d146c8b">

Reply:
Ah ok, got it. Then let's name it `source` or `channel`